tests/testthat/test_load_data.R

```diff
@@ -0,0 +1,107 @@
+# WHDH --------------------------------------------------------------------
```

Comment:
Please add a test for the handling of legacy xmart calls

Reply:
Calls to xMart are avoided in the unit tests as pulling data from xMart takes a while and having those tests run every time we run `check()` would cause significant delays. Once the xMart part of the new `load_billion_data` is working (which I plan on making much more optimised), I will add tests for that.